### ISSUE TITLE
Make bootloader using mbed-cmake

### ISSUE THREAD

**Issue by deddy11** [USCRPL/mbed-cmake#34]
I try to make bootloader by using mbed-cmake. I follow [this](https://github.com/ARMmbed/mbed-os-example-bootloader) tutorial and find out that mbed-cmake can not produce [project-name]_application.bin, only [project-name].bin.

Does it mean that I can't use compiled bootloader binary file from mbed-cmake to be merge with application binary file?
Please help me. Thanks.

**Comment by multiplemonomials** [comment]
Yeah, currently mbed-cmake doesn't support bootloaders.  Might add it sometime in the future though...

**Comment by deddy11** [comment]
I see... thanks for your fast response
